rfcs/0031-devices-api.md

```diff
@@ -127,7 +127,7 @@ This simple wrapper enables type checking of these functions and defining a clea
 [reference-level-explanation]: #reference-level-explanation
 
 ## Entrypoint
-The entrypoint API defined in [Embedded C Runtime Interface](https://discuss.tvm.apache.org/t/rfc-utvm-embedded-c-runtime-interface/9951) is augmented with the `devices` structure which contains implemented `tvm_device_t` `struct`s for each device used by the network:
+The entrypoint API defined in [Embedded C Runtime Interface](https://discuss.tvm.apache.org/t/rfc-utvm-embedded-c-runtime-interface/9951) is augmented with the `tvm_mynetwork_devices` structure which contains implemented `tvm_device_t` `struct`s for each device used by the network:
```

Comment:
i forget if we discussed this already, but @manupa-arm 's RFC used a different casing so let's just quickly address this here: google c++ styleguide would have us call this `TvmGenMynetworkDevices`. I kind of like the `_` separator however, since this name is generated.

Reply:
I've intentionally left this as the C++ style as that's the current standard in TVM, @areusch suggested changing it [would likely require an RFC in itself and updating the style guide](https://github.com/apache/tvm/pull/8280#issuecomment-888554759).